test/test_gpio.c

```diff
@@ -4,35 +4,41 @@
 
 #include "unity.h"
 
-void setup_test(void) { setbuf(stdout, NULL); }
+void setup_test(void) {}
 
 void teardown_test(void) {}
 
 // gpio_init
 
 // Tests that the GPIO init works. This should simply return true 100% of time unless the test is on
 // x86 and the configuration is incorrect in which case fix it!
-void test_gpio_init(void) { TEST_ASSERT_TRUE(gpio_init()); }
+void test_gpio_valid_init(void) { TEST_ASSERT_TRUE(gpio_init()); }
```

Comment:
Based on the partitioning of your unit tests, maybe it would make more sense to have test_[module]_[function]_test?
Up to you though, it doesn't really matter to me.